### DISCUSSION TITLE
租赁GPU云服务器是否能提高算力节省训练时间？

### DISCUSSION THREAD

**Discussion by happysunguy841031** [PaddlePaddle/PaddleOCR#13891] [Q&A] [answered]
大家好，不好意思又来叨扰了。

目前小弟用了1张4060显卡在第一阶段（标记模型）训练 3W+ 的数据图片（epoch_num 3000）就已经需要 3个月+ 的时间。代表如果后续增加数据量训练时间只会增加不会减少，不晓得能不能透过租赁GPU服务器缩减训练时长或有没有办法可以加速。

谢谢指教。

**Comment by TOMORINAONAO** [comment]
需要3000个epoch这么多么，我也不是很专业，不过我自己更改字典训练也只是选择50epoch和100epoch。我是简繁体合一起训练，12000的字典，1000多w的数据，50个epoch准确率有85左右。我的场景下绝大部分都可以识别准确，个别涉及字母或符号的会出错。

**Reply by happysunguy841031** [reply]
第一次尝试使用训练，所以对参数的拿捏比较没有底。可以向您请教您的数据量大概在多少张？每张标记的字数是多少吗？谢谢您。

**Reply by TOMORINAONAO** [reply, marked as answer]
> 第一次尝试使用训练，所以对参数的拿捏比较没有底。可以向您请教您的数据量大概在多少张？每张标记的字数是多少吗？谢谢您。

  首先分场景，我的场景是英文、中文简体、中文繁体，然后是包含横排和竖排文字，所以我准备了六种数据集。每一种都准备了一二百万张，每行的字数从五个字到三十来字的都有（我的目标场景是文档场景识别，所以文字长度参考了常见文档长度，图片背景颜色同理）。
  paddleocr官方也给过参考意见说每一个字符出现要在200次以上，同时要平衡各字符的比例不要差太多。
  我个人的经验是训练出来的模型识别文字效果都很好，标点符号的识别相比官方模型有所下降，然后就在合成数据的时候额外多加了很多符号。

**Reply by TOMORINAONAO** [reply]
忘了说了我只训练了识别模型，检测模型在我的场景下已经够用了就没去额外微调。另外我训练的前几轮是使用所有数据训练的，训练了几轮之后使用ratio_list参数控制每轮数据量只有总数据的百分之二三十左右，我实际的总数据量是有点多的，单张4090要是用千万级的数据量完整训练也得一两个月了。
你可以去aistudio上看看别人分享的项目有没有关于检测模型的训练，我识别模型的参数就是参考上面一个手写体训练的项目里的配置。

**Reply by happysunguy841031** [reply]
收到，谢谢您百忙中抽空回复我。
